### PR TITLE
[ALLUXIO-2441] Disable packet streaming by default

### PR DESCRIPTION
**Pull request by peisun1115**
https://alluxio.atlassian.net/browse/ALLUXIO-2441

Disable packet streaming by default

### CODE REVIEW

**Comment by peisun1115** [comment]
@calvinjia

**Comment by calvinjia** [comment]
Could you add docs for this configuration value?

**Comment by peisun1115** [comment]
@calvinjia i've added docs, PTAL?

File: docs/_data/table/en/user-configuration.yml [hunk 1]
```diff
@@ -85,3 +85,5 @@ alluxio.user.ufs.file.writer.class:
   Selects networking stack to run the client with for writing to under file system through a
   worker's data server. Currently only `alluxio.client.netty.NettyUnderFileSystemFileWriter`
   (remote write using netty) is valid.
+alluxio.user.packet.streaming.enabled:
+  If set to true, the packet streaming data transfer protocol is used.
```
Comment (calvinjia) [inline docs/_data/table/en/user-configuration.yml]: Could you label this as experimental and add a note about the possible benefits?

Comment (peisun1115) [inline docs/_data/table/en/user-configuration.yml]: Done

**Pull request merged**